app/views/admin/editions/_show_metadata.html.erb

```diff
@@ -45,6 +45,33 @@
 
 <%= render_partial_if_exists 'extra_metadata' %>
 
+<% if @edition.can_be_tagged_to_taxonomy? && tagging_taxonomy_enabled? %>
+  <section class="taxonomy-topics" id="topic-new-taxonomy">
+    <h2>Topics (new taxonomy)
+      <%= link_to edit_admin_edition_tags_path(@edition.id), class:"btn btn-default pull-right" do %>
+        <% if @expanded_links.selected_taxon_paths.any? %>
+          <span class="glyphicon glyphicon-edit"></span> Change topics
+        <% else %>
+          <span class="glyphicon glyphicon-plus-sign"></span> Add topic
+        <% end %>
+      <% end %>
+    </h2>
+    <% if @expanded_links.selected_taxon_paths.any? %>
+      <div class="content content-bordered">
+        <% @expanded_links.selected_taxon_paths.each do |path| %>
+          <%= render partial: 'taxon_breadcrumb', locals: {
+            breadcrumbs: path
+          } %>
+        <% end %>
+      </div>
+    <% else %>
+      <div class="no-content no-content-bordered">
+        No topics - you must add a topic before publishing
```

Comment:
Perhaps `you must` -> `please`? We'll change that back after we've made it mandatory.